src/server/index.spec.tsx

```diff
@@ -303,17 +304,25 @@ describe(__filename, () => {
       });
 
       it('tightens style-src, script-src and img-src if CDN URL is unset', async () => {
+        const fakeEnv = {
+          ...prodEnv,
+          REACT_APP_REVIEWERS_HOST: 'https://reviewers.addons-dev.allizom.org',
```

Comment:
Why not adding this env variable to the `prodEnv` object?

Reply:
Good idea!